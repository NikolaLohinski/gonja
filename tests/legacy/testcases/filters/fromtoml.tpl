{{ "[root.nested.section]\nvalue = true" | fromtoml | tojson }}