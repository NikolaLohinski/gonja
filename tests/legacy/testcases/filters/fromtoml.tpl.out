{"root":{"nested":{"section":{"value":true}}}}